employee-rostering-frontend/cypress/plugins/index.js

```diff
@@ -1,19 +1,3 @@
-/*
```

Comment:
What happened to the license?

Reply:
Because those were generated files - we're not the author => no need to license them. The license was only added to make ESLint happy. In my opinion it's better to disable ESLint completely for those folders (until we start adding code there) but keep the generated examples because:
- they're good examples, they might spark a new ideas when facing Cypress configuration issues,
- they would be generated on each Cypress run so we need to either gitignore them or check them in. The second option is better in my opinion.

Reply:
We added lines there (`require('cypress-failed-log');`) and ```
  on('task', {
    failed: require('cypress-failed-log/src/failed')(),
  })
```

Reply:
(those lines show actions when a cypress task fail in the log)